classes/ValidateConstraintTranslator.php

```diff
@@ -49,14 +49,32 @@ public function __construct(TranslatorInterface $translator)
      */
     public function translate($validator)
     {
-        if ($validator === 'isName' || $validator === 'isCustomerName') {
+        if ($validator === 'isName') {
             return $this->translator->trans(
                 'Invalid name',
                 [],
                 'Shop.Forms.Errors'
             );
         }
 
+        if ($validator === 'isCustomerName') {
+            return $this->translator->trans(
+                'Invalid name',
+                [],
+                'Shop.Forms.Errors'
+            ) . PHP_EOL .
+            $this->translator->trans(
+                'Invalid characters: 0-9!<>,;?=+()@#"°{}_$%/\^*`',
+                [],
+                'Shop.Forms.Errors'
+            ) . PHP_EOL .
+            $this->translator->trans(
+                'A space is required after . and 。',
```

Comment:
Shouldn't it be written with quotation marks? Example: _A space is required after "." and "。"_

Reply:
I don't know what is the best practice.

Reply:
Tristan was for the ""

Reply:
So you mean using `'A space is required after "." and "。"'`
(Isn't the space after 。 weird in this case?)

Reply:
NB: the space can't be removed, it's part of the character

Reply:
@matks told us it is the equivalent of "." in China and that you can use it as a dot in a URL

Reply:
But yeah you are right the space is part of it so I don't really understand how it can be used for a URL 🤔

Reply:
I think maybe some email browsers simplify them and convert them into regular `.` thus creating a clickable url
But my question was is it ok if we have this little space inside the `""` ?

Reply:
@jolelievre, as for me, it is!

Reply:
Great, then @PierreRambaud can you re-approve this PR please? ^^

Reply:
> I think maybe some email browsers simplify them and convert them into regular `.` thus creating a clickable url
> But my question was is it ok if we have this little space inside the `""` ?
I confirm, try visiting https://www.apple。com 😄

Reply:
But... this PR has been merged without the fix on the wording?

Reply:
> Great, then @PierreRambaud can you re-approve this PR please? ^^

@LouiseBonnard we've been tricked !

Reply:
Oups my bad, I was too quick and thought I had already added it
Here's a PR to fix this ^^
https://github.com/PrestaShop/PrestaShop/pull/14163